src/index.d.ts

```diff
@@ -1879,9 +3498,60 @@ declare namespace admin.database {
       committed: boolean,
       snapshot: admin.database.DataSnapshot|null
     }>;
+
+    /**
+     * Writes multiple values to the Database at once.
+     *
+     * The `values` argument contains multiple property-value pairs that will be
+     * written to the Database together. Each child property can either be a simple
+     * property (for example, "name") or a relative path (for example,
+     * "name/first") from the current location to the data to update.
+     *
+     * As opposed to the `set()` method, `update()` can be use to selectively update
+     * only the referenced properties at the current location (instead of replacing
+     * all the child properties at the current location).
+     *
+     * The effect of the write will be visible immediately, and the corresponding
+     * events ('value', 'child_added', etc.) will be triggered. Synchronization of
+     * the data to the Firebase servers will also be started, and the returned
+     * Promise will resolve when complete. If provided, the `onComplete` callback
+     * will be called asynchronously after synchronization has finished.
+     *
+     * A single `update()` will generate a single "value" event at the location
+     * where the `update()` was performed, regardless of how many children were
+     * modified.
+     *
+     * Note that modifying data with `update()` will cancel any pending
+     * transactions at that location, so extreme care should be taken if mixing
+     * `update()` and `transaction()` to modify the same data.
+     *
+     * Passing `null` to `update()` will remove the data at this location.
+     *
+     * See
+     * {@link
+     *  https://firebase.googleblog.com/2015/09/introducing-multi-location-updates-and_86.html
+     *  Introducing multi-location updates and more}.
+     *
+     * @example
+     * ```javascript
+     * var adaNameRef = admin.database().ref('users/ada/name');
+     * // Modify the 'first' and 'last' properties, but leave other data at
+     * // adaNameRef unchanged.
+     * adaNameRef.update({ first: 'Ada', last: 'Lovelace' });
+     * ```
+     *
+     * @param values Object containing multiple values.
+     * @param onComplete Callback called when write to server is
+     *   complete.
+     * @return Resolves when update on server is complete.
+     */
     update(values: Object, onComplete?: (a: Error|null) => any): Promise<void>;
   }
 
+  /**
+   * @extends {admin.database.Reference}
+   * @extends {admin.Thenable<void>}
```

Comment:
There's no such thing as `admin.Thenable` as far as I know. Lets remove that line.